src/commands/stats.js

```diff
@@ -6,8 +6,44 @@ module.exports = {
    * Displays the server stats.
    * @param {Discord.Message} message the message provided by discord
    */
-  command: function stats(message) {
+  command: function stats(message, client) {
     try {
+      let totalSeconds = client.uptime / 1000;
+      const days =
+        Math.floor(totalSeconds / 86400) <= 0
+          ? ''
+          : Math.floor(totalSeconds / 86400) < 10
+          ? `0${Math.floor(totalSeconds / 86400)} days`
+          : `${Math.floor(totalSeconds / 86400)} days`;
+
+      const hours =
+        Math.floor(totalSeconds / 3600) <= 0
+          ? ''
+          : Math.floor(totalSeconds / 3600) < 10
+          ? `0${Math.floor(totalSeconds / 3600)} hours`
+          : `${Math.floor(totalSeconds / 3600)} hours`;
+
+      totalSeconds %= 3600;
+      const minutes =
+        Math.floor(totalSeconds / 60) <= 0
+          ? ''
+          : Math.floor(totalSeconds / 60) < 10
+          ? `0${Math.floor(totalSeconds / 60)}  minutes`
+          : `${Math.floor(totalSeconds / 60)} minutes`;
+
+      const seconds =
+        Math.floor(totalSeconds % 60) < 10
+          ? `0${Math.floor(totalSeconds % 60)} seconds`
+          : `${Math.floor(totalSeconds % 60)} seconds`;
+
+      const date = new Date();
+      const hour =
+        date.getHours() < 10 ? `0${date.getHours()}` : date.getHours();
+      const mins =
+        date.getMinutes() < 10 ? `0${date.getMinutes()}` : date.getMinutes();
+      const tz = /\((.*)\)/.exec(new Date().toString())[1];
+      const am_pm = hour >= 12 ? 'PM' : 'AM';
```

Comment:
Move the 2 distinct code blocks to their own functions that return the strings provided to the `statsEmbed` block.

Reply:
I'll do this tomorrow. But I'll look into luxon and maybe even use it.

Reply:
Moved code blocks into each corresponding function that will return string that contains each corresponding data.